docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -24,29 +28,117 @@ docker run -p 8080:8080 feedhenry/raincatcher-keycloak
 
 For more information about the *{Keycloak}* Docker Image, see the *{Keycloak}* link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/README.md[README.md].
 
+IMPORTANT: The {Keycloak} realm setup included in this docker image is for demo purposes only. Ensure that you provide proper client 
```

Comment:
IMHO Best to say that is configured only for local development. 
Demo purposes will suggest that someone may run that on openshift or somewhere else, but configuration in the image will not allow that.